vite.config.js

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react()]
```

Comment:
It could be interesting to take a look at the Vite Community plugins curated at [awesome-vite](https://github.com/vitejs/awesome-vite#plugins) in case anything there catches your eye. Vite also has a testing framework called [`vitest` ](https://github.com/vitest-dev/vitest) that you might be interested in taking a look at.